style.css

```diff
@@ -82,7 +81,188 @@
   color: blue;
 }
 
-@media (max-width: 600px) {
+.header h1 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  margin-top: 50px;
+}
+
+.line hr {
+  display: none;
+  background: #36b37e;
+  width: 10%;
+  height: 5px;
+  border-radius: 24px;
+  margin-left: 45%;
+  margin-top: 40px;
+}
+
+.grid {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  margin: 30px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  background: #ebf0ee;
+}
+
+.grid1 {
+  height: 300px;
+  border-radius: 5px;
+}
+
+.grid2 {
+  background: #fff;
+  border-radius: 10px;
+  width: 100%;
+}
+
+.grid2 h2 {
+  color: #3a4a42;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+}
+
+.grid2 ul {
+  display: flex;
+  list-style: none;
+  justify-content: center;
+  gap: 20px;
+}
+
+.grid2 li {
+  border: 1px solid #ebf0ee;
+  background: #ebf0ee;
+  border-radius: 4px;
+  padding: 5px;
+  color: #3a4a42;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+}
+
+.grid2 button {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  border-radius: 5px;
+  border: 1px solid #36b37f;
+  background: #36b37f;
+  width: 125px;
+  height: 50px;
+  margin-bottom: 30px;
+}
+
+.abt-main h1 {
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
```

Comment:
- [ ] While you did a good job styling your project :+1:, as stated by the previous reviewer kindly fix the design because some of the elements are overlapping as you can see below

**Your design**

![Screenshot from 2023-02-15 07-22-46](https://user-images.githubusercontent.com/105117832/219028074-fe1b13ba-6088-45df-8efd-2f473a8c989f.png)

**Template**

![Screenshot from 2023-02-15 07-32-15](https://user-images.githubusercontent.com/105117832/219028310-dc5d8fb8-f88e-4a4b-bee4-ad019b8e7a78.png)